test/Oscoin/Test/Storage/Block/Equivalence.hs

```diff
@@ -25,6 +21,11 @@ import           Oscoin.Test.Crypto.Blockchain.Generators
 import           Oscoin.Test.Storage.Block.SQLite (DummySeal, defaultGenesis)
 import           Oscoin.Test.Util (Condensed(..), showOrphans)
 
+import           Data.ByteArray.Orphans ()
+import qualified Data.List as List
+import qualified Data.Text as T
+import           GHC.Exception
```

Comment:
What do we need the `GHC.Exception` import for? 🤔

Reply:
`srcLocFile` and `srcLocStartLine` apparently

Reply:
Made them explicit :+1: